i18n/de.json

```diff
@@ -2672,7 +2672,7 @@
   "user.settings.advance.deactivateDesc": "Das Deaktivieren ihres Kontos entfernt ihre Möglichkeit, sich an diesem Server anzumelden und deaktiviert alle mobilen und E-Mail-Benachrichtigungen. Kontaktieren Sie ihren Systemadministrator, um ihr Konto zu reaktivieren, ",
   "user.settings.advance.deactivateDescShort": "Klicken Sie auf 'Bearbeiten', um ihr Konto zu deaktivieren",
   "user.settings.advance.enabledFeatures": "{count, number} {count, plural, one {Funktion} other {Funktionen}} aktiviert",
-  "user.settings.advance.formattingDesc": "Wenn aktiviert, werden Nachrichten so formatiert, dass Links erstellt, Emojis angezeigt, Text formatiert und Zeilenumbrüche hinzugefügt werden. Standardmäßig ist dies aktiviert. Ändern der Einstellung erfordert ein Neuladen der Seite.",
+  "user.settings.advance.formattingDesc": "If enabled, posts will be formatted to create links, show emoji, style the text, and add line breaks. By default, this setting is enabled.",
```

Comment:
Was this intended to replace in English?
Same with others.

Reply:
This string was recently updated - so it shows up in English until the translators have updated to the new translation @saturninoabril

Reply:
Oops, sorry for delayed review.  I missed this.